gift_compassion/models/contracts.py

```diff
@@ -77,15 +79,40 @@ def open_gifts(self):
         }
 
     def hold_gifts(self):
-        """ Postpone open gifts. """
+        for contract in self:
+            today = datetime.date.today()
+            first_day = today.replace(day=1)
+            last_day = today.replace(day=calendar.monthrange(today.year,
+                                                             today.month)[1])
+            suspended_gifts = self.env['sponsorship.gift'].search([
+                ('child_id.project_id', '=', contract.project_id.id),
+                ('state', '=', 'open'),
```

Comment:
Still not sure it will do what we want... The gifts in open state are those sent but not yet received by GMC. There should not be a lot of them. You should instead search for "In Progress" gifts (or both)